demo/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "demo",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "build": "gatsby build",
+    "clean": "gatsby clean",
+    "dev": "ENABLE_GATSBY_REFRESH_ENDPOINT=true gatsby develop"
```

Comment:
Is `ENABLE_GATSBY_REFRESH_ENDPOINT=true` needed for this demo?

Reply:
@notrab No, not 'needed'. But equally not interfering.